uint8 system_id
uint8 component_id
time timestamp
string main_mode
string sub_mode
bool armed
string autopilot
uint8 base_mode
string mav_state
string mav_type
uint8 mavlink_version